pkg/registry/servicecatalog/broker/strategy_test.go

```diff
@@ -61,3 +94,57 @@ func TestBroker(t *testing.T) {
 		t.Fatalf("Fresh broker should have empty status")
 	}
 }
+
+func TestValidateUpdateStatusPrepareForUpdate(t *testing.T) {
+	cases := []struct {
+		name                string
+		old                 *sc.Broker
+		newer               *sc.Broker
+		shouldChecksum      bool
+		checksumShouldBeSet bool
+	}{
+		{
+			name:                "not ready -> not ready",
```

Comment:
I know we do not do it for the other tables, and I do not like that either. 

It is not immediately clear what each field is doing and how they get checked later. Each time we follow this pattern there should be a comment explaining what the setup is.